src/inputs/dropdown/dropdown.jsx

```diff
@@ -157,6 +162,7 @@ class Dropdown extends React.Component {
                     const value = {
                         id: itemId || null,
                         label: itemLabel,
+                        value: itemValue,
```

Comment:
Do we need these changes here?

Reply:
It ain't critical but according to the DOCS we can pass the values for `<Dropdown />` items only through the `options` prop, It's not possible to pass a value using the `<Dropdown.Item />` children component. Anyway, now that the component is deprecated it doesn't make much sense, I'm reverting it back.